### DISCUSSION TITLE
I want to run material for mk docs on HTTPS

### DISCUSSION THREAD

**Discussion by syedshadab19** [squidfunk/mkdocs-material#4245] [Q&A] [answered]
Hi I want to run the material for mkdocs on https  can you please specify the command for it and also what do I need to change in the docker file?

**Comment by squidfunk** [comment, marked as answer]
The built-in development server of MkDocs is [__not intended for production use__](https://github.com/mkdocs/mkdocs/issues/1946#issuecomment-569113058). You should invoke `mkdocs build` and then serve the resulting static site with another production-grade server (or use a service like GitHub Pages).

**Reply by syedshadab19** [reply]
Hi @squidfunk  can you ellaborate little bit more on how to make it work.

**Reply by squidfunk** [reply]
Our [publishing guide](https://squidfunk.github.io/mkdocs-material/publishing-your-site/) includes recipes for GitHub Pages and GitLab Pages, both free of charge. If you want to self-host, you can use any production-grade web server, e.g. [nginx](https://www.nginx.com/).

**Reply by syedshadab19** [reply]
ok thank you
